code/script.js

```diff
@@ -1,39 +1,211 @@
+    <option value="Trolls World Tour">Trolls World Tour</option>
+    <option value="VIDIYO™">VIDIYO™</option>
+    <option value="Xtra">Xtra</option>
+  </select>
+`;
+	const select = document.getElementById('select');
+	select.addEventListener('change', () => evaluateThemeSelection(select.value));
+};
+
+// This function will handle the LEGO theme choice made by the user 
+const evaluateThemeSelection = (selectedTheme) => {
+	if (selectedTheme === "Star Wars™") {
+    showMessage(selectedTheme, 'user');
+		starWarsItems();
+	} else {
+    showMessage(selectedTheme, 'user');
+    setTimeout(() => {showMessage(`Shoot! ${selectedTheme} is out of stock. How about Star Wars™ lego?`, 'bot');
+    inputWrapper.innerHTML = `
+    <button id="yesBtn">Ok! Show me what you got in stock!</button>
+    <button id="notInterestedBtn">No, not interested.</button>
+    `;
+    document.getElementById('yesBtn').addEventListener('click', () => { 
+      showMessage(`Ok. Show me what you got in stock!`, 'user');
+      starWarsItems();
+    });
+    document.getElementById('notInterestedBtn').addEventListener('click', () => handleNotInterested());}, 1000);
+	}
+};
+
+
+// This function will present the LEGO Star Wars items available for order
+const starWarsItems = () => {
+  setTimeout(() => {showMessage(`May the force be with you, you made the only valid choice 🥳 ! Choose one of our top picks:`, 'bot');
+	inputWrapper.innerHTML = `
+  <button id="btn1">Millenium Falcon</button>
+  <button id="btn2">R2D2</button>
+  <button id="btn3">Death Star</button>
+    `;
+	document.getElementById('btn1').addEventListener('click', () => confirmOrder('Millenium Falcon'));
+	document.getElementById('btn2').addEventListener('click', () => confirmOrder('R2D2'));
+	document.getElementById('btn3').addEventListener('click', () => confirmOrder('Death Star'));}, 1000);
+};
+
+
+// This function will handle the not interested reply 
+const handleNotInterested = () => {
+  showMessage(`No, not interested.`, 'user');
+  goodByeMessage();
+}
+
+// This function will ask the user to confirm the purchase 
+const confirmOrder = (item) => {
+	showMessage(item, 'user');
+	setTimeout(() => {showMessage(`Excellent choice! One ${item} have now been added to your cart. Are you happy with your order?`, 'bot');
+	inputWrapper.innerHTML = `
+  <button id="confirmBtn">Yes!</button>
+  <button id="notConfirmedBtn">No!</button>
+  `;
+  document.getElementById('confirmBtn').addEventListener('click', () => thankYou(item));
+  document.getElementById('notConfirmedBtn').addEventListener('click', () => {
+    showMessage('No.', 'user');
+    goodByeMessage();
+});}, 1000);
+}
+
+// This function will close the deal and end the chat with Thank You message and order confirmation. 
+const thankYou = (item) => {
+  showMessage('Yes!', 'user');
+  setTimeout(() => {showMessage(`Awesome! One ${item} will be sent to you from a galaxy far far away.`, 'bot');
+  playSound ();
```

Comment:
This was amazing. I did jump when the music came on the first time, sneaky with the little delay.

Reply:
Agree! With headphones it was loud! But very entertaining